integration-test/helpers/harness.js

```diff
@@ -17,40 +17,29 @@ const setup = async (ops) => {
     const loadExtension = ops.loadExtension !== false
     const tmpDirPrefix = path.join(os.tmpdir(), DATA_DIR_PREFIX)
     const dataDir = fs.mkdtempSync(tmpDirPrefix)
+    /** @type {import('puppeteer').PuppeteerLaunchOptions} */
     const puppeteerOps = {
-        args: [
-            `--user-data-dir=${dataDir}`
-        ],
         headless: 'chrome'
     }
+    const args = [
+        `--user-data-dir=${dataDir}`
+    ]
 
     if (loadExtension) {
         let extensionPath = 'build/chrome/dev'
         if (process.env.npm_lifecycle_event === 'test-int-mv3') {
             extensionPath = extensionPath.replace('chrome', 'chrome-mv3')
         }
-        puppeteerOps.args.push('--disable-extensions-except=' + extensionPath)
-        puppeteerOps.args.push('--load-extension=' + extensionPath)
+        args.push('--disable-extensions-except=' + extensionPath)
+        args.push('--load-extension=' + extensionPath)
     }
 
     // github actions
     if (process.env.CI) {
-        puppeteerOps.args.push('--no-sandbox')
-    }
-
-    if (process.env.TRAVIS) {
```

Comment:
Deleted this travis code because we don't use it and was invalid to the latest puppeteer anyway.